### ISSUE TITLE
Reply to, linking to a post

### ISSUE THREAD

**Issue by 0gust1** [vole/vole#62]
It would be neat to have functionnalities around conversation : linking to a post, reply-to a post ... Maybe the post's hash/id could be used as an HTML id attribute value ?

**Comment by marktheunissen** [comment]
Thanks, yes we have plans for these. The hash / permalink idea is interesting, hadn't thought of that.

**Comment by rupl** [comment]
I agree, this would be awesome. Could we have the Vole server provide single-post JSON at an endpoint? something simple like `/post/[uuid]`? While it's technically a separate issue, linking to other users could be done in the same fashion with `/user/[uuid]` and instead of an HTML ID for user we just create a data attribute.

Perhaps we could have a modal for this instead of linking to an anchor, since pagination and other factors mean it's very likely that replies won't be on the same page. Once we have a robust framework maybe try inline threads a la Twitter.
